NuKeeper.Inspection.Tests/Sources/NugetConfigFileParserTests.cs

```diff
@@ -0,0 +1,102 @@
+using NuKeeper.Inspection.Sources;
+using NUnit.Framework;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace NuKeeper.Inspection.Tests.Sources
+{
+    [TestFixture]
+    public class NugetConfigFileParserTests
+    {
+        [Test]
+        public void EmptyStringReturnsNull()
+        {
+            var data = string.Empty;
+
+            var parser = new NugetConfigFileParser(new NullNuKeeperLogger());
+
+            var sources = parser.Parse(ToStream(data));
+
+            Assert.That(sources, Is.Null);
+        }
+
+        [Test]
+        public void InvalidStringReturnsNull()
+        {
+            var data = "not valid markup";
+
+            var parser = new NugetConfigFileParser(new NullNuKeeperLogger());
+
+            var sources = parser.Parse(ToStream(data));
+
+            Assert.That(sources, Is.Null);
+        }
+
+        [Test]
+        public void ValidDataWithOneItemReturnsThatItem()
+        {
+            var data =
+@"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+  <packageSources>
+    <add key=""NuGet official"" value=""https://nuget.org/api/v2/"" />
```

Comment:
I know it's the default at your $work, and this is just test data, but it's a bit odd to see v2 here :D

Reply:
Yep, i should change that.